packages/react-internal/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -138,7 +139,10 @@ export const PersonaCoinBase: React.FunctionComponent<IPersonaCoinProps> = React
             <div
               className={mergeStyles(
```

Comment:
this is not directly related to your change. but we shouldn't be calling `mergeStyles` here without memoizing.

@TristanWatanabe - could you make **a separate PR** to extract this `mergeStyles` call to avoid calling `mergeStyles` on every render (which leads to bad perf), something like:

adding this at file scope:
```js=
const getInitialsStyles = memoizeFunction((className, primaryText, text, initialsBackgroundColor, initialsTextColor) => mergeStyleSets(
  className,
  { 
    backgroundColor: getPersonaInitialsColor(primaryText, text, initialsBackgroundColor),
    color: initialsTextColor
  }
));
```

then calling `getInitialsStyles` here:
```js=
className={getInitialsStyles(
```
instead of 
```js=
className={mergeStyles(
```

you will need to update the signature of  `getPersonaInitialsColor` to take only the args it needs, not the entire set of `props`